check_cron_or_pr.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-if [[ $github.event.number ]]; then
+if [[ $github.event_name == "pull_request" ]]; then
```

Comment:
I think this should be [`GITHUB_EVENT_NAME`](https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables) instead, since we're in the shell here.

Reply:
Ya, I'm still learning... Thx

Reply:
Thx It's working.